serve/graph/schema/types/block.graphql

```diff
@@ -25,9 +31,91 @@ type Block {
   """
   time: Time!
 
+  """
+  The number of transactions this Block belongs to.
+  """
+  num_txs: Int!
+
+  """
+  The total number of transactions that have occurred up to this block.
+  """
+  total_txs: Int!
```

Comment:
what is the difference between `total_txs` and `num_txs`? It is not clear to me from the documentation.

Reply:
I added the following explanation. 
Is there any ambiguity?
```
The total number of transactions that have occurred up to this block.
  Indicates the total number of transactions that have occurred up to this point, even if there are no transactions in this block.
```

commit: https://github.com/gnolang/tx-indexer/pull/56/commits/bbc5645ab9950e40ed89cc73e5fb3faeab5b75ac